src/components/GroupedGoalList.tsx

```diff
@@ -70,8 +70,17 @@ export const GroupedGoalList: React.FC<GroupedGoalListProps> = ({ queryState, se
         return pages.flatMap((page) => page.projects);
     }, [data]);
 
+    const handleItemEnter = useCallback(
+        (goal: GoalByIdReturnType) => {
+            if (goal) {
```

Comment:
Why this logic not in onKeyboardClick? Do we really need call callback if argument and current element are undefined?

Reply:
fixed

Reply:
nope